components/map/map.js

```diff
@@ -293,6 +293,18 @@ function Map({commune, isAddressFormOpen, handleAddressForm}) {
         </Pane>
       )}
 
+      {hint && (
+        <Pane
+          zIndex={1}
+          position='fixed'
+          width='400px'
+          top={140}
+          left='55%'
```

Comment:
Je propose : 
- de supprimer le `width` car il n'est pas obligatoire, la fenêtre s'adapte à la taille du message.
- de mettre `top` à 130px pour être aligné avec les contrôles de la carte et le bouton de fermeture du menu.
- Utiliser `align-self` pour ne pas avoir de nombre magique 

```suggestion
          zIndex={1}
          position='fixed'
          alignSelf='center'
          top={130}
```